src/app/home/HomeComponent.tsx

```diff
@@ -41,18 +64,43 @@ const HomeComponent: React.FunctionComponent<IHomeComponentProps> = ({
       </NavList>
     </Nav>
   );
+  const isWelcomeScreen = history.location.pathname === '/welcome';
+  const [isShowNav, setIsShowNav] = useState(false);
+
+  useEffect(() => {
+    if (isWelcomeScreen) {
+      setIsShowNav(false);
+    }
+  }, [isWelcomeScreen]);
+
+  const onNavToggle = () => {
+    setIsShowNav(!isShowNav);
+  };
 
   return (
     <Page
-      header={<PageHeaderComponent showNavToggle />}
-      sidebar={<PageSidebar nav={nav} theme="dark" />}
-      isManagedSidebar
+      header={
+        <PageHeaderComponent
+          onNavToggle={onNavToggle}
+          showNavToggle={!isWelcomeScreen}
+          setNamespaceSelectIsOpen={setNamespaceSelectIsOpen}
+          isWelcomeScreen={isWelcomeScreen}
+        />
+      }
+      sidebar={<PageSidebar nav={nav} isNavOpen={!isWelcomeScreen} theme="dark" />}
+      isManagedSidebar={isShowNav}
       skipToContent={<SkipToContent href={`#${mainContainerId}`}>Skip to content</SkipToContent>}
       mainContainerId={mainContainerId}
     >
+      <ActiveNamespaceModal namespaceSelectIsOpen={namespaceSelectIsOpen} />
+
       <Switch>
         <Route exact path="/">
-          <Redirect to="/clusters" />
+          {/* NATODO: Are we showing the welcome screen for all users? */}
+          {isHideWelcomeScreen ? <ClustersPage /> : <Redirect to="/welcome" />}
```

Comment:
I don't think we should render `<ClustersPage>` directly here, since that means there are two URLs for it (`/` and `/clusters`). We should probably just render the `<Redirect to="/clusters" />` that we had before, so `/` is purely for redirecting

Reply:
Resolved in 5b4c2a018c93384715eefd213f1da2660403e35b